source/common/json/config_schemas.cc

```diff
@@ -832,6 +832,45 @@ const std::string Json::Schema::FAULT_HTTP_FILTER_SCHEMA(R"EOF(
   }
   )EOF");
 
+const std::string Json::Schema::IP_TAGGING_HTTP_FILTER_SCHEMA(R"EOF(
+  {
+    "$schema": "http://json-schema.org/schema#",
+    "type" : "object",
+    "properties" : {
+      "request_type" : {
+        "type" : "string",
+        "enum" : ["internal", "external", "both"]
+      },
+      "ip_tags" : {
+        "type" : "array",
+        "minItems" : 1,
+        "items" : {
+          "type" : "object",
+          "properties" : {
+            "ip_tag_name" : {
+              "type" : "string"
+            },
+            "ip_list" : {
+              "type" : "array",
+              "minItems" : 1,
+              "uniqueItems": true,
+              "items" : {
+                "type": "string",
+                "anyOf": [
+                  { "format" : "ipv4" },
```

Comment:
on adding this test it looks like we are not actually enforcing this format. I verified with tests that exercise this https://github.com/lyft/envoy/blob/master/source/common/json/config_schemas.cc#L79. The schema validator lets the wrong string through, and it is envoy code that catches it. Specifically: https://github.com/lyft/envoy/blob/master/source/common/network/utility.cc#L28